docs/feature_combo.md

```diff
@@ -7,7 +7,6 @@ To enable this feature, you need to add `COMBO_ENABLE = yes` to your `rules.mk`.
 Additionally, in your `config.h`, you'll need to specify the number of combos that you'll be using, by adding `#define COMBO_COUNT 1` (replacing 1 with the number that you're using).
 <!-- At this time, this is necessary -->
 
-Also, by default, the tapping term for the Combos is set to the same value as `TAPPING_TERM` (200 by default on most boards). But you can specify a different value by defining it in your `config.h`.  For instance: `#define COMBO_TERM 300` would set the time out period for combos to 300ms.
 
 Then, your `keymap.c` file, you'll need to define a sequence of keys, terminated with `COMBO_END`, and a structure to list the combination of keys, and it's resulting action.
```

Comment:
This kinda goes beyond the committed changes but while we're editing the combo docs, we may as well correct that sentence. It should be 
« Then, **in** your `keymap.c` file, you'll need define a sequence of keys, terminated with `COMBO_END`, and a structure to list the combinations of keys, and **its** resulting action. » 
(bolded text emphasizes additions/modifications, they shouldn't be added to the docs)